internal/operators/lvm/common.go

```diff
@@ -0,0 +1,67 @@
+package lvm
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+)
+
+type baseHardware struct {
+	CPUCores int64 `json:"cpu_cores"`
+	RAMMib   int64 `json:"ram_mib"`
+}
+
+type hardwareRequirement struct {
```

Comment:
any reason why we shouldn't use [this](https://github.com/openshift/assisted-service/blob/master/internal/hardware/versioned-requirements.go)?